server/dao/dao.js

```diff
@@ -43,4 +46,8 @@ exports.findProducts = (searchString, category) =>
 exports.createProductsTextSearchIndexes = () => {
   createProductsTextSearchIndexes(db);
 };
+
 exports.deleteEmployee = (employeeID) => deleteEmployee(db, employeeID);
+
+exports.createOrder = (clientId, products, status, totalPrice) =>
```

Comment:
use clientID instead of clientId (everywhere else ID is uppercase, it is lowercase only in the case it's just "id")